pyproject.toml

```diff
@@ -1,26 +1,25 @@
 [build-system]
-requires = ["hatchling"]
-build-backend = "hatchling.build"
-
-[tool.hatch.build.targets.wheel]
-packages = ["src/pyromb"]
-
-[tool.hatch.build]
-only-packages = true
-exclude = [
-  ".conda",
-]
+requires = ["setuptools>=61.0.0", "wheel"]
+build-backend = "setuptools.build_meta"
 
 [project]
 name = "pyromb"
-version = "0.2.0"
+version = "0.2.2"
 authors = [ 
-    { name="Tom Norman", email="tom@normcosystems.com" }
+    { name = "Tom Norman", email = "tom@normcosystems.com" }
 ]
 description = "Runoff Model Builder (Pyromb) is a package used for building RORB and WBNM control files from catchment diagrams built from ESRI shapefiles. Its primary use is in the QGIS plugin Runoff Model: RORB and Runoff Model: WBNM"
 readme = "README.md"
 requires-python = ">=3.9"
 
+dependencies = [
+    "shapely",
```

Comment:
Remove Shapely and Pyshp dependency and use GDAL. GDAL is packaged with QGIS so will be one less dependecy to install.